cmd/clairctl/.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+go:
+- 1.5
```

Comment:
I think that to should be better to integrate it in the main .travis file.
No?

Le mar. 3 mai 2016 17:49, Quentin Machu <notifications@github.com> a écrit :

> In cmd/clairctl/.travis.yml
> <https://github.com/coreos/clair/pull/164#discussion_r61905322>:
>
> > @@ -0,0 +1,19 @@
> > +language: go
> > +go:
> > +- 1.5
>
> We should do tip and 1.6 as well if possible. This is non-blocker though,
> we can do it later.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/coreos/clair/pull/164/files/0cf009041a3af8b1f1503dfae0f06f0ed428555a#r61905322>
>